juniper_official/Solutions/EVPN-VXLAN/vtep-interface-count.rule

```diff
@@ -29,133 +86,63 @@ healthbot {
-                value 10;
-                description "VTEP threshold";
+            variable vtep-threshold-max {
+                value 3200;
+                description "Enter VTEP interface max threshold";
                 type int;
             }
             vector endpoint-list {
                 path "/topic[topic-name=protocol.evpn]/rule[rule-name=check-vtep-interface]/endpoint-address";
-                time-range 30s;
-            }
-            rule-properties {
-                version 1;
-                contributor juniper;
-                supported-healthbot-version 2.1.0;
-                catalogue {
-                    tier 2;
-                }
-                supported-devices {
-                    juniper {
-                        operating-system junos {
-                            products QFX {
-                                releases 18.1R1 {
-                                    release-support min-supported-release;
-                                    platform All;
-                                }
-                            }
-                        }
-                    }
-                }
-                helper-files other {
-                    list-of-files VtepTable.yml;
-                }
-            }
-        }
-        rule check-vtep-interface {
-            keys vtep-name;
-            synopsis "VTEP interface count KPI";
-            description "Display all the VTEP interfaces";
-            sensor vtep-sensor {
-                synopsis "iAgent sensor definition";
-                iAgent {
-                    file VtepTable.yml;
-                    table VtepTable;
-                    frequency 30s;
-                }
-            }
-            field endpoint-address {
-                sensor vtep-sensor {
-                    path vtep-address;
-                }
-                type string;
-                description "Sensor field to store end point address.";
-            }
-            field endpoint-type {
-                sensor vtep-sensor {
-                    path vtep-type;
-                }
-                type string;
-                description "Sensor field to store end point type.";
-            }
-            field vtep-name {
-                sensor vtep-sensor {
-                    path name;
-                }
-                type string;
-                description "Sensor field to store end point name.";
-            }
-            trigger VTEP-Endpoint {
-                synopsis "VTEP interface  KPI";
-                description "Display all VTEP interfaces";
-                frequency 30s;
-                term Term_1 {
-                    then {
-                        status {
-                            color green;
-                            message "Interface : $vtep-name Endpoint Type: $endpoint-type Endpoint Address: $endpoint-address";
-                        }
-                    }
-                }
+                time-range 60s;
```

Comment:
Increase rule time-range from 60 to 65s as refrred rule also has same freq.

Reply:
updated the rule